operator/config/crd/bases/forklift.konveyor.io_forkliftcontrollers.yaml

```
@@ -62,6 +62,9 @@ spec:
               feature_ocp_live_migration:
                 type: string
                 enum: ["true", "false"]
+              feature_ova_appliance_management:
+                type: string
+                enum: [ "true", "false" ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)